app/views/attachment_masks/done.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= _('Attachment available for download.') %></h1>
+
+<p>
+  <%= _('The attachment you\'ve requested has now been processed and is ' \
+        'available for download. The processed attachment will be retained ' \
+        'for up to 7 days. Once 7 days has lasped then the cache might be ' \
+        'cleared again.') %>
```

Comment:
Where's the 7 days coming from here? I don't think _Alaveteli_ purges attachments (though we've manually hacked this in in the case of some installs we host). I'm not sure we'll need to do that with this setup, so maybe making this  as simple as:

```
<%= _('The attachment has now been processed and is available for download.') %>
```

should be fine. Also worth setting the `@title` instance variable.

Reply:
I'm not sure, I will re-word.